google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineFacetConfiguration.java

```diff
@@ -50,17 +58,33 @@ public void readExternal(Element element) throws InvalidDataException {
   public void writeExternal(Element element) throws WriteExternalException {
   }
 
+  public Set<AppEngineStandardMavenLibrary> getLibraries(@NotNull Project project) {
+    LibraryTable libraryTable = LibraryTablesRegistrar.getInstance().getLibraryTable(project);
+    Library[] libraries =  libraryTable.getLibraries();
+
+    Set<AppEngineStandardMavenLibrary> configuredLibraries = Sets.newHashSet();
+    for (Library configuredLibrary : libraries) {
+      AppEngineStandardMavenLibrary mavenLibrary
+          = AppEngineStandardMavenLibrary.getLibraryByMavenDisplayName(configuredLibrary.getName());
+      if (mavenLibrary != null) {
```

Comment:
I added it in another location: AppEngineSupportProvider#addMavenLibraries - I think I mistook your comment for that class.

In this case, a warning log is not correct: if the mavenLibrary is null, this just means that the project configured library does not match one of our AE managed libraries and we skip it. This is a normal condition and a warning is not necessary.